packages/playground/website/src/lib/tracking.ts

```diff
@@ -1,23 +1,94 @@
+import { StepDefinition } from '@wp-playground/blueprints';
+
 /**
  * Declare the global window.gtag function
  */
 declare global {
-    interface Window { gtag: any; }
+	interface Window {
+		gtag: any;
+	}
 }
 
 /**
  * Google Analytics event names
  */
-type GAEvent = 'load' | 'step';
+type GAEvent = 'load' | 'step' | 'install' | 'error';
 
 /**
  * Log a tracking event to Google Analytics
  * @param GAEvent The event name
  * @param Object Event data
  */
-export const logTrackingEvent = (event: GAEvent, data?: {[key: string]: string}) => {
-    if (typeof window === 'undefined' || !window.gtag) {
-        return;
-    }
-    window.gtag('event', event, data);
-}
+export const logTrackingEvent = (
+	event: GAEvent,
+	data?: { [key: string]: string }
+) => {
+	if (typeof window === 'undefined' || !window.gtag) {
+		return;
+	}
+	window.gtag('event', event, data);
```

Comment:
What do you think of adding a try/catch here? If there are ever any tracking errors, it seems like they should not break Playground. If we do that, maybe we can try to log a brief error to Slack if tracking fails so it doesn't fail silently.

Reply:
> we can try to log a brief error to Slack if tracking fails so it doesn't fail silently.

I just did a warning. That Slack channel isn't great as-is today. We don't monitor it actively and errors are only sent if a user decides to send it from the Error modal. 

Long-term, yes I would like to send all these errors somewhere and have alerting.

Reply:
> I just did a warning. That Slack channel isn't great as-is today. We don't monitor it actively and errors are only sent if a user decides to send it from the Error modal.

Makes sense. Thanks!